### PR TITLE
Adding option to get the model full predictions on the test set

### PR DESCRIPTION
**Pull request by lperryp**
## Status
- [] In Progress
- [ x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Adding option to get the model full predictions on the test set

### CODE REVIEW

**Comment by lperryp** [comment]
@yaron-libman hi, can you check the comment at: Scripts/DBotTrainTextClassifierV2/CHANGELOG.md? thanks

File: Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py [hunk 1]
```diff
@@ -283,6 +262,48 @@ def validate_data_and_labels(data, exist_labels_counter, labels_mapping, missing
         return_error('\n'.join(err))
 
 
+def return_file_result_with_predictions_on_test_set(data, original_text_fields, test_index, text_field, y_test, y_pred_dict):
+    if original_text_fields is None or original_text_fields.strip() == '':
+        original_text_fields = [text_field]
+    else:
+        original_text_fields = re.split(r'[|,]', original_text_fields)
```
Comment (erezh31) [inline Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py]: strip each term (sometimes they use spaces)

File: Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py [hunk 2]
```diff
@@ -283,6 +262,48 @@ def validate_data_and_labels(data, exist_labels_counter, labels_mapping, missing
...
+    predictions_data = {}
```
Comment (erezh31) [inline Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py]: add the tokenized text field in any case, even if the origianl text fields specified

**Comment by richardbluestone** [comment]
I've reviewed and updated the md file.

**Comment by richardbluestone** [comment]
Files have been reviewed as requested.

**Pull request merged**